README.md

```diff
@@ -103,6 +103,13 @@ In fact, `ltt` is `pip` with a few added options:
   ltt install --pytorch-computation-backend=cu102 torch
   ```
 
+  Borrowing from the mutex packages that PyTorch provides for `conda` installations,
+  `--cpuonly` is available as shorthand for `--pytorch-computation-backend=cu102`.
```

Comment:
Driveby, because I realized that `--cpuonly` was not mentioned here, although I use it regularly.